Presence of ` /etc/apt/apt.conf.d/zzzz-temp-installer-unattended-upgrade` causes verification to fail

## Description

While doing QA for the 2.11.0 release, I discovered that the verification step was failing with:

```
FAILED common/test_automatic_updates.py::test_unattended_upgrades_functional[paramiko:/mon] - assert 'Allowed origins are: origin=Ubuntu,archive=focal, origin=Ubuntu,archive=focal-security, origin=Ubuntu,archive=focal-updates, origin=SecureDrop,codename=focal'
```

whereas I had:

```
o=Ubuntu,a=focal, o=Ubuntu,a=focal-security, o=UbuntuESMApps,a=focal-apps-security, o=UbuntuESM,a=focal-infra-security, origin=Ubuntu,archive=focal, origin=Ubuntu,archive=focal-security, origin=Ubuntu,archive=focal-updates, origin=SecureDrop,codename=focal\nInitial
```

After tracking this down, we discovered a file named `zzz-temp-installer-unattended-upgrade` in `/etc/apt/apt.conf.d` that was adding the additional ESM origins, causing the test to fail.

This is [an upstream Ubuntu bug](https://bugs.launchpad.net/subiquity/+bug/2053002)

## Steps to Reproduce

I'm not sure how to reproduce - my app server was installed on the same day using the same installation media and the exact same steps as the mon server, but the file only existed on mon.

## Expected Behavior

`./securedrop-admin verify` completes without issue.

## Actual Behavior

`./securedrop-admin verify` fails due to the unexpected sources.

## Comments

@legoktm mentioned adding a step to our Ansible playbook that checks for and removes the file if it exists.

--- a/securedrop/debian/securedrop-config.postinst
+++ b/securedrop/debian/securedrop-config.postinst
@@ -31,6 +31,10 @@ case "$1" in
     # Migrate the ssh group to sdssh
     securedrop-migrate-ssh-group.py
 
+    # Should not exist but might because of an Ubuntu installer bug that is
+    # now fixed: https://bugs.launchpad.net/subiquity/+bug/2053002
+    rm -f /etc/apt/apt.conf.d/zzzz-temp-installer-unattended-upgrade
+
     ;;
     abort-upgrade|abort-remove|abort-deconfigure)
     ;;

